docs/services/liveinternet.rst

```diff
@@ -0,0 +1,60 @@
+==================================
+Liveinternet -- traffic analysis
+==================================
+
+
+`Liveinternet`_ is an analytics tool like as google analytics.
+
+.. _`Liveinternet`: https://www.liveinternet.ru/code/
+
+
+.. yandex-metrica-installation:
+
+Installation
+============
+
+To start using the Liveinternet integration, you must have installed the
+django-analytical package and have added the ``analytical`` application
+to :const:`INSTALLED_APPS` in your project :file:`settings.py` file.
+See :doc:`../install` for details.
+
+Next you need to add the Liveinternet template tag to your templates.
+
+The Liveinternet counter code is inserted into templates using a template
+tag.  Load the :mod:`liveinternet` template tag library and insert the
+:ttag:`liveinternet` tag.  To display as a single image combining a counter
+and the LiveInternet logo::
+
+    {% load liveinternet %}
+    <html>
+    <head>
+    ...
+    {% liveinternet %}
+    </head>
+    ...
+In the form of two images, one of which is a counter (transparent GIF size 1x1),
```

Comment:
I believe, there should be an empty line to separate the code block from this text block.